int16 BatteryStatus 
int32 BatteryVoltage
int32 ChargingCurrent
bool ChargingFlag
int16 Temperature
int16 RemainCapacity
int16 FullCapacity
